<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QWidget" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>预览与坐标设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="posSettingsBtn">
           <property name="text">
            <string>位置设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>提示语：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tipWord"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>背景图片：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectImageBtn">
           <property name="text">
            <string>选择图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>目标日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="targetDate">
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>记录窗口位置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="isRecPosition">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,0,10,0,1,0,10">
       <item>
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>[提示语]字体样式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>字体颜色：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tipFontColor"/>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>例：255,255,255</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>字体：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFontComboBox" name="tipFontStyle"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>字体大小：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tipFontSize"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>字体加粗程度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tipFontBoldLevel"/>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>注:为0不加粗</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>[天数]字体样式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>字体颜色：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dayFontColor"/>
           </item>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>例：255,255,255</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>字体：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFontComboBox" name="dayFontStyle"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>字体大小：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="dayFontSize"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>字体加粗程度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="dayFontBoldLevel"/>
           </item>
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>注:为0不加粗</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0,10,0,1,0,10">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>[当前日期]字体样式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>字体颜色：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="curDayFontColor"/>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>例：255,255,255</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>字体：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFontComboBox" name="curDayFontStyle"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>字体大小：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="curDayFontSize"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>字体加粗程度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="curDayFontBoldLevel"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>注:为0不加粗</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>[目标日期]字体样式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>字体颜色：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tarDayFontColor"/>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>例：255,255,255</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>字体：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFontComboBox" name="tarDayFontStyle"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>字体大小：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tarDayFontSize"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>字体加粗程度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tarDayFontBoldLevel"/>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>注:为0不加粗</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyBtn">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
